modmesh/pilot/_base_app.py

```diff
@@ -293,17 +293,38 @@ def editable(self, row, col):
         """
         Check if the cell is editable.
 
-        :param row: Row index.
-        :type row: int
-        :param col: Column index.
-        :type col: int
+        :param row (type: int): Row index.
```

Comment:
Please keep [the original sphinx format for argument type](https://www.sphinx-doc.org/en/master/tutorial/describing-code.html#documenting-python-objects) (`:type row: int`).